.vscode/launch.json

```diff
@@ -1,38 +1,44 @@
 {
-    // Use IntelliSense to learn about possible attributes.
-    // Hover to view descriptions of existing attributes.
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-    "version": "0.2.0",
-    "configurations": [
-        {
-            "name": "Development",
-            "request": "launch",
-            "type": "dart",
-            "program": "lib/main_development.dart",
-            "args": [
-                "--flavor",
-                "development"
-            ]
-        },
-        {
-            "name": "Staging",
-            "request": "launch",
-            "type": "dart",
-            "program": "lib/main_staging.dart",
-            "args": [
-                "--flavor",
-                "staging"
-            ]
-        },
-        {
-            "name": "Production",
-            "request": "launch",
-            "type": "dart",
-            "program": "lib/main_production.dart",
-            "args": [
-                "--flavor",
-                "production"
-            ]
-        },
-    ]
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Development",
+      "request": "launch",
+      "type": "dart",
+      "program": "lib/main_development.dart",
+      "args": [
+        "--flavor",
+        "development",
+        "--dart-define",
+        "API_BASE_URL=https://randomuser.me"
```

Comment:
Not sure if it's the correct place to do it, but I propose to migrate to `dart-defines-from-file`?

```
        "--dart-define-from-file",
        "dart_defines.json"
```

Reply:
That will be nice. Didn't know it existed! 😄